pkg/logger/log/accesslog.go

```diff
@@ -237,7 +237,11 @@ func getRequestBody(req *restful.Request, contentType string) string {
 		if strings.Contains(contentType, "application/json") {
 			return util.MinifyJSON(bodyBytes)
 		}
-		return string(bodyBytes)
+
+		bodyString := string(bodyBytes)
```

Comment:
whether have to do this replaceAll in the endpoint request, how's the performance downgrade?

Reply:
Hi, I have done quick testing for the `ReplaceAll` performance.
From the result, the `ReplaceAll` will not giving too much performance downgrade. From test result below, the average of `ReplaceAll` execution time is only < 2 ms.

Another notes: Most of the time, our endpoints will use Content-Type `application/json` which will go into this condition: `return util.MinifyJSON(bodyBytes)`.

This PR is to cover for non-`application/json` Content-Type, e.g. `text/plain`.

Testing codes:

```
func Test_ReplaceAll_Performance(t *testing.T) {
	str := "4KB OF STRING"
	i := 0
	for i < 10000 {
		start := time.Now()
		strings.ReplaceAll(str, "\n", "\\n")
		strings.ReplaceAll(str, "\r", "\\r")
		fmt.Printf("Took %v\n", time.Since(start))
		i++
	}
}
```

Testing result:

```
Took 3.53µs
Took 1.777µs
Took 1.566µs
Took 1.776µs
Took 1.512µs
Took 9.368µs
Took 1.506µs
Took 1.526µs
Took 1.473µs
Took 1.436µs
Took 8.357µs
Took 1.562µs
Took 1.484µs
Took 1.45µs
Took 1.499µs
Took 4.527µs
Took 1.605µs
Took 1.636µs
Took 1.473µs
Took 1.484µs
Took 2.801µs
Took 3.3µs
Took 1.653µs
Took 1.649µs
Took 1.641µs
Took 3.693µs
Took 2.699µs
Took 1.456µs
Took 1.506µs
Took 1.473µs
Took 3.304µs
Took 1.567µs
Took 1.552µs
Took 1.445µs
Took 1.492µs
Took 2.769µs
Took 1.495µs
Took 1.59µs
Took 2.423µs
Took 1.551µs
Took 2.651µs
Took 1.737µs
Took 1.621µs
Took 1.711µs
Took 1.527µs
Took 2.576µs
Took 1.513µs
Took 1.541µs
Took 1.624µs
Took 1.532µs
Took 3.52µs
....
....
....
Took 1.341µs
Took 708ns
Took 2.793µs
Took 853ns
Took 2.93µs
Took 810ns
Took 714ns
Took 723ns
Took 735ns
Took 2.839µs
Took 786ns
Took 752ns
```